lib/erb_lint/linters/hard_coded_string.rb

```diff
@@ -63,7 +63,7 @@ def autocorrect(processed_source, offense)
         string = offense.source_range.source
         return unless (klass = load_corrector)
         return unless string.strip.length > 1
-        node = RuboCop::AST::StrNode.new(:str, [string])
+        node = ::RuboCop::AST::StrNode.new(:str, [string])
```

Comment:
What's the reason for explicitly fully qualifying the `Rubocop` references?

Reply:
The first thing I wanted to check in the code was all dependencies on `RuboCop`. Given that `erb_lint` has a class called `Rubocop` (no capital 'C' though), this wasn't perfectly clear at first so I thought that it was a good idea to do this, instead of relying on the context or how the letter 'c' is capitalized.

Reply:
I'm 👍 with fully qualifying external references (in general), but especially if there's a similarly named thing internally.